BaseFilter/sections/specific.txt

```diff
@@ -16962,8 +16962,8 @@ outlook.live.com#?#.customScrollBar[data-is-scrollable] > div[class] > div[class
 ! check-host.net
 check-host.net##div[id*="r"][id*="d"][id*="c"] img
 check-host.net##.text-left a > img[loading="lazy"]
-check-host.net#?#a[href]:contains(/protection|сайт/)
-check-host.net#?#div + div[class*=" "]:last-child:contains(/^ ?[×✕✖X☓χx⨯✗\+\＋\﹢хХΧẊẋ] ?$/):upward(1)
+check-host.net##a[href*="prometon.net"]
+check-host.net#?#div + div[class*=" "]:last-child:contains(/^.*[×✕✖X☓χx⨯✗\+\＋\﹢хХΧẊẋ].*$/):upward(1)
```

Comment:
The rule is too wide because of using `.*`. I will change it a bit.

Reply:
Your rule breaks the site.
"https://check-host.net/ip-info?host=https://1.1.1.1"

Reply:
I tried doing the same thing but saw it broke the site, which is why I went with `.*`

Reply:
My rule did not break anything from what I could tell.

Reply:
Well, don't worry - I've changed the rule. Thanks for letting me know about this.